wireapp/wire-ios-mono
# feat: see e2ei certificate details UI WPB-5524

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-5524" title="WPB-5524" target="_blank"><img alt="Sub-task" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10816?size=medium" />WPB-5524</a>  [iOS] 5.3.25 See E2EI certificate details - UI
  </td></table>
  <br />
 




































----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?
This PR shows certificate if available from the Device details screen
### Issues
NA

### Causes (Optional)



### Solutions

Shows a way to display and download E2EI certificate

### Dependencies (Optional)

_If there are some other pull requests related to this one (e.g. new releases of frameworks), specify them here._

Needs releases with:


### Testing

#### Test Coverage (Optional)


#### How to Test

Check for testing E2EI steps in the developer menu and make sure there is a valid state which could show a certificate. 

### Notes (Optional)


### Attachments (Optional)
![simulator_screenshot_C9F818DF-393A-4DB7-BB09-536D5DA2538F](https://github.com/wireapp/wire-ios/assets/17357739/3624d6d6-ac3e-44aa-8653-0c221fff82f7)

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
